### ISSUE TITLE
Incorporating Helm Chart into syseleven-exporter Repository

### ISSUE THREAD

**Issue by baurmatt** [vfm/syseleven-exporter-chart#12]
Hi vfm 👋,

My name is Matthias Baur, and I currently hold the position of Principal Engineer at SysEleven. In our recent efforts to enhance the syseleven-exporter, my team and I have taken over maintenance from Staffbase. We've already released a new version yesterday, adding support for OpenStack application credentials.

As part of our efforts, we're considering integrating the Helm Chart into the syseleven-exporter repository to provide a more streamlined experience for our users. Before moving forward, we wanted to reach out to you and discuss the possibility of collaborating on this integration.

Would you be interested? :) 

Best regards,
Matthias Baur

https://github.com/syseleven/syseleven-exporter

**Comment by baurmatt** [comment]
Hey @rriverak @wmvfm @rdxmb, I've seen you originally created the repository. Do you have an opinion on my request? :)

**Comment by rdxmb** [comment]
@baurmatt Although I did not create the repository:

Having this supported directly by SysEleven would be perfect. For me as a customer, it is actually the only way to have our object-storage-quota monitored.

In the past, I've pimped the Grafana-Board to have multiple projects included, see https://github.com/vfm/syseleven-exporter-chart/issues/9 . Are you interested in those?

**Comment by jenshavelberg** [comment]
Hi there,

Thank you for reaching out and for your interest in this repository.

We would like to inform you that we have made the decision to archive this repository. This decision comes as a direct consequence of its parent project by Staffbase being archived two years ago. Since the main project is no longer active or maintained, we believe it is appropriate to also cease further development and maintenance of this extension.

However, we understand that this tool may still hold value for the community. Therefore, we are open to SysEleven taking over ownership of this repository. If SysEleven is interested in incorporating the exporter into their GitHub organization, they are welcome to do so and continue its maintenance.

We appreciate your understanding and support throughout the project's lifecycle. Should SysEleven or any other maintainer wish to take over the repository, please let us know, and we will make the necessary arrangements for the transition.

Best regards,
Jens

**Comment by baurmatt** [comment]
@jenshavelberg, thanks for you answer and sorry for not calling you back! We incorporated the Helm chart in our fork of the syseleven-exporter. We've also published a new version to https://syseleven.github.io/syseleven-exporter.

@rdxmb Sounds interesting 🤔 Would it be integrable into the existing dashboard? We'd like to avoid maintaining multiple dashboards :) Feel free to open a issue/PR in our project -> https://github.com/syseleven/syseleven-exporter
